MSProject/ViewControllers/LibraryTableView/ViewModel/WBLibraryViewModel.swift

```diff
@@ -31,35 +31,37 @@ class WBLibraryViewModel {
     }
     
     var numberOfCells: Int {
-        return bookViewModels.count
+        return bookViewModels.value.count
     }
 
     func getCellViewModel(at indexPath: IndexPath) -> WBBookViewModel {
-        return bookViewModels[indexPath.row]
+        return bookViewModels.value[indexPath.row]
     }
     
-    func selectBook(at indexPath: IndexPath) -> WBBookViewModel {
-        return bookViewModels[indexPath.row]
+    func loadBooks() -> SignalProducer<[WBBook], RepositoryError> {
+        return self.repository.getBooks().on(failed: { [unowned self] _ in self.bookViewModels = MutableProperty([]) }, value: { [unowned self] value in
+            self.bookViewModels = MutableProperty(value.map { WBBookViewModel(book: $0) })
```

Comment:
you don't need to assign a new MutableProperty, you can change the .value property of bookViewModels